webapp/src/main/resources/config/application.properties

```diff
@@ -286,4 +286,7 @@ spring.session.jdbc.table-name=SPRING_SESSION_V2
 
 springdoc.swagger-ui.csrf.enabled=true
 
-server.servlet.session.cookie.name=JSESSIONID
\ No newline at end of file
+server.servlet.session.cookie.name=JSESSIONID
+
+l10n.ai.openai.host=http://127.0.0.1:19193/
+l10n.ai.service.type=OpenAI
```

Comment:
These two parameters should also not be present in the packaged application.properties